kore/src/Kore/Step/Simplification/Exists.hs

```diff
@@ -253,8 +282,10 @@ makeEvaluateBoundRight variable freeSubstitution normalized = do
     TopBottom.guardAgainstBottom simplifiedPattern
     return simplifiedPattern
   where
+    simplifiedPattern :: Pattern variable
     simplifiedPattern =
-        Conditional.andCondition
+        Pattern.markSimplified
```

Comment:
Does this eliminate the need to `markSimplified` in `quantifyPattern` below? How do we know all parts of the pattern are really simplified?

Reply:
You're right, we don't, I'll fix that.

Reply:
Fixed.